src/utils/app.utils.ts

```diff
@@ -0,0 +1,68 @@
+import { EventName, EventStatus, StatusMessages } from 'src/common/constants';
+import { FormattedData } from 'src/common/types/webhook.event.types';
+
+export function isPayloadValid(payload: any): boolean {
+  return (
+    payload?.action &&
+    payload?.pull_request &&
+    payload?.repository &&
+    payload?.sender &&
+    payload?.installation
+  );
```

Comment:
PullProofAI

The `isPayloadValid` function checks for specific properties in the payload object. It would be beneficial to add type annotations to improve code readability and catch potential errors during development.

```diff
+ export function isPayloadValid(payload: { action: any, pull_request: any, repository: any, sender: any, installation: any }): boolean {
```

#### Issue:


<!-- This is an auto-generated comment by PullProofAI -->